src/routes/api/installation/api-installation-get.ts

```diff
@@ -25,30 +25,31 @@ export const ApiInstallationGet = async (req: Request, res: Response): Promise<v
 			subscriptions.map(async (subscription) => {
 				const id = subscription.gitHubInstallationId;
 				try {
-					const response = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_INSTALLATION_API) ?
-						await gitHubAppClient.getInstallation(id) :
-						await client.apps.getInstallation({ installation_id: id });
+					const response: AxiosResponse<Octokit.AppsGetInstallationResponse> = await gitHubAppClient.getInstallation(id);
 
-					response.data.syncStatus = subscription.syncStatus;
-					return response.data;
+					return {
+						...response.data,
+						isGlobalInstall: response.data.repository_selection === "all",
```

Comment:
How about `isGlobalInstall: response.data?.repository_selection === "all",`?
As this is ran before the `installations.filter(!response.error)`, so in case of error, I am not sure whether there still be `data` in the `response`. 
ps: I do understand the try/catch in this function, still feels weird. If above `null exception` throw, in the error acch the `errr.status` won't be 404, blha .... I don't know,

Reply:
Nevermind, seems okay, hmm.

Reply:
Sure, will do!